okhttp-tls/src/main/java/okhttp3/tls/HandshakeCertificates.kt

```diff
@@ -67,20 +67,49 @@ import javax.net.ssl.X509TrustManager
  *    roots private to an organization or service.
  */
 class HandshakeCertificates private constructor(
-  private val keyManager: X509KeyManager,
-  private val trustManager: X509TrustManager
+  @get:JvmName("keyManager") val keyManager: X509KeyManager,
+  @get:JvmName("trustManager") val trustManager: X509TrustManager
```

Comment:
I think they don't have to be `private` given that we have `fun keyManager(): X509KeyManager`.